src/content/api/patron-codes.mdx

```diff
@@ -0,0 +1,203 @@
+---
+title: Patron Codes
+description: Patron Code model and related endpoints
+draft: true
+nav:
+  path: API
+  order: 31
+---
+import Properties from '../../components/Properties.astro';
+import Property from '../../components/Property.astro';
+import Error from '../../components/Error.astro';
+import Endpoint from '../../components/Endpoint.astro';
+import CodePanel from '../../components/CodePanel.astro';
+
+A Patron Code is an alternative to presenting a QR code where that option isn’t available.
+
+## Model
+
+### Attributes
+
+<Properties>
+  <Property name="id" type="string">
+    The Patron Code's unique identifier.
+  </Property>
+
+  <Property name="barcode" type="string">
+    ✩ Field to display in barcode format.
+  </Property>
+
+  <Property name="createdAt" type="timestamp">
+    Date when the Patron Code was created.
+  </Property>
+
+  <Property name="expiresAt" type="timestamp">
+    Date when the Patron Code will expire.
+  </Property>
+
+  <Property name="appName" type="string" deprecated>
+    The app used to create the Patron Code.
+  </Property>
+
+  <Property name="accountId" type="string">
+    The account id of the creator of the patron code.
+  </Property>
+
+  <Property name="testScenarioName" type="string">
+    The [Test Scenario Name](#test-scenario-name) of the Patron Code.
+  </Property>
+</Properties>
+
+✩ Barcode is a 16 digit number. The first 6 digits are a Centrapay defined prefix, then a 9 digit
+code, then a luhn checksum digit.
+
+```
```

Comment:
Suggestion: This should be in a CodePanel component so it's uniform with other examples.